src/NServiceBus.Persistence.AzureStorage/Config/AzureSagaPersisterConfig.cs

```diff
@@ -1,52 +0,0 @@
-namespace NServiceBus.Config
-{
-    using System.Configuration;
-
-    /// <summary>
-    /// Config section for the Azure Saga Persister
-    /// </summary>
-    public class AzureSagaPersisterConfig:ConfigurationSection
```

Comment:
Thx for reminding me of this @SeanFeldman . Although I've been seeing v1 in docs it slipped my mind when I was writing the comment above. I agree with you though, that users of the previous package may just want to copy the configuration over.